packages/terra-spacer/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+ChangeLog
+=========
+
+0.0.0 - (<!--- Insert date using June 28, 2017 format. --->)
```

Comment:
Change this to unreleased.

Reply:
@jmsv6d updated https://github.com/cerner/terra-core/pull/1011/commits/51079e4e1f7a6e3bb384bba084478bb7fcf12335#diff-f5728bf897114026571ceed082d4bcfeR3